mne/time_frequency/tfr.py

```diff
@@ -1930,6 +1930,32 @@ def __isub__(self, tfr):  # noqa: D105
         self.data -= tfr.data
         return self
 
+    def __truediv__(self, a):  # noqa: D105
+        return self.__div__(a)
+
+    def __div__(self, a):  # noqa: D105
```

Comment:
I don't think we need `__div__` anymore now that we use Python3. I would remove this and put it all under `__truediv__`